tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -309,32 +309,30 @@ Re-validate License For Disabled Application From Enabled Page
    ${buttons_here}=  Get WebElements    xpath://div[contains(@class,'popover__body')]//button[text()='here']
    Click Element  ${buttons_here}[0]
 
-
-
-
 Get Question Mark Links
-    [Documentation]  It returns the link elements from the question mark
-    @{links_list} =  Create List
+    [Documentation]      It returns the link elements from the question mark
+    @{links_list}=  Create List
     @{link_elements}=  Get WebElements    //a[@class="odh-dashboard__external-link pf-c-dropdown__menu-item" and not(starts-with(@href, '#'))]
```

Comment:
robobop : 315:1 [W] 0508 Line is too long.
How can I fix it ?

Reply:
```
@{link_elements}=  Get WebElements
...    //a[@class="odh-dashboard__external-link pf-c-dropdown__menu-item" and not(starts-with(@href, '#'))]
```
Something like this?

Reply:
It worked